src/extension.ts

```diff
@@ -76,8 +73,15 @@ export async function activate(context: vscode.ExtensionContext) {
     }),
   );
 
-  registerUpdateHistoryLinksCommand(context, () => {
+  registerUpdateHistoryLinksCommand(context, (testID?: string) => {
     historyProvider.refresh();
+
+    if (testID) {
+      const testRecord = storage.getTestRecord(testID);
+      if (testRecord) {
```

Comment:
Hm. `getTestRecord` will always return a test record in the successful case. If it isn't successful, there'll be an error thrown. So I don't think this check here does anything?